### ISSUE TITLE
Subsection CD, definition of "minor": it's more general than that

### ISSUE THREAD

**Issue by darijgr** [rbeezer/fcla#85]
You write that "However, informally, a minor is a determinant of a submatrix, specifically det(A(i|j))". This is not quite true: the submatrix doesn't have to be of the form A(i|j), but can be of any size from "empty" to "whole A" (as long as it is square). Not a big issue for a text that doesn't actually use the terminology, but wanted to have this mentioned anyway.

**Comment by rbeezer** [comment]
Yes, I could firm this up.  I think I once thought of not even discussing this.

"minor of an entry" or "first minor" might be a better way to describe what I 
meant here.
